mk/spksrc.service.create_links

```diff
@@ -5,9 +5,9 @@ service_create_links ()
     for cmd in ${SPK_COMMANDS}
     do
         if [ -e "${SYNOPKG_PKGDEST}/${cmd}" ]; then
-            mkdir -p "$(dirname /usr/local/${cmd})"  >> "${INST_LOG}" 2>&1
-            echo "create link: /usr/local/${cmd} -> ${SYNOPKG_PKGDEST}/${cmd}"  >> "${INST_LOG}"
-            ln -s "${SYNOPKG_PKGDEST}/${cmd}" "/usr/local/${cmd}"  >> "${INST_LOG}" 2>&1
+            mkdir -p "$(dirname /usr/local/${cmd})" 2>&1 | install_log
```

Comment:
Can we remove the duplicate redirections trough out?

Reply:
> Maybe we can use install_log to encapsulate all package scripts but allow a switch or function to disable it in case a package wants to notify the user. IMHO logs should always be written and not something a package author should worry about. 

I like the idea to always write to the installer log file and it is easy to implement.
But I did not find any solution on how to disable this and write to stdout while all output is captured for the installer log (I tried to collect notification output in a global variable or to use file descriptor 3 - nothing worked).

The only solution I can imagine for user notification (i.e. writing to stdout) is to implement additional service functions that are not redirected to the installer log like `validate_preinst` or `validate_preupgrade` and that must be used for such validations.
This would work at least for validation of wizard variables, but not for complex installers that have multiple installation steps with extra validation (if we have any).
The real benefit of user notification is for `preinst`, `preuninst` and `preupgrade` anyway, as the user can navigate back in the wizard and fix the configuration before starting the installation/uninstallation.
Further a `validate_postinst` could notify when additional manual configuration might be required.